tt_metal/tt_metal.cpp

```diff
@@ -423,11 +423,11 @@ void WriteToDeviceInterleavedContiguous(const Buffer &buffer, const std::vector<
         buffer.size());
 
     uint32_t page_size = buffer.page_size();
-    TT_FATAL(buffer.size() % page_size == 0);
+    TT_FATAL(buffer.size() % page_size == 0, "Buffer size {} mod page_size {} needs to be 0.", buffer.size(), page_size);
```

Comment:
suggesting rewording 
```suggestion
    TT_FATAL(buffer.size() % page_size == 0, 
             "Invalid buffer size: {}. Buffer size must be a multiple of page size {}.", 
             buffer.size(), page_size);

```

Reply:
Updating wording for the three cases.